caexporter.go

```diff
@@ -43,7 +48,8 @@ type Collector struct {
 }
 
 func init() {
-	prometheus.MustRegister(version.NewCollector(namespace))
+	prometheus.MustRegister(version.NewCollector(namespace), scrape_error)
+	scrape_error.Set(1)
```

Comment:
I don't think you should set scrape error on init, it should be the opposite imo.